### ISSUE TITLE
The tool is crashing

### ISSUE THREAD

**Issue by khansaad1275** [Whomrx666/Email-scraper#1]
![image](https://github.com/Whomrx666/Email-scraper/assets/32749921/05c7e61c-ac35-45d3-bbf2-372b0c151e02)

**Comment by Whomrx666** [comment]
yeah i dont know why some features and modules in termux is error in 2024
not like before you can install its in google cloudshell or kali linux its
work thats why i dont install in in my termux in the main page pictures

Pada Kam, 1 Feb 2024 pukul 15.51 Saad Maqsood ***@***.***>
menulis:
